manageiq-operator/pkg/helpers/miq-components/httpd.go

```diff
@@ -176,6 +177,18 @@ func addOIDCEnv(secretName string, podSpec *corev1.PodSpec) {
 	podSpec.Containers[0].Env = append(podSpec.Containers[0].Env, clientSecret)
 }
 
+func configureHttpdOIDCEnv(podSpec *corev1.PodSpec, managed_client_id string, managed_client_secret string, managed_app_domain string, managed_mgmt_domain string) {
+	managedCfgClientId := corev1.EnvVar{Name: "MANAGED_CFG_CLIENT_ID", Value: managed_client_id}
```

Comment:
I think we should we be using `ValueFrom` here instead to pull the ENV var from the secret rather than creating a clear text ENV var with the data. (I'm assuming that some of this is sensitive data, like `managed_client_secret`)

Reply:
Yeah, I had ValueFrom originally, however, Kubernetes does not see that as a change when the value is updated in the secret, so the Httpd pod will not restart. Those values accessible to an admin as they exist in the /etc/httpd/user-conf.d/oidc.conf file.

Reply:
It's clear text to anyone that can `kubectl describe pod httpd-zzzzz` as well.

Reply:
@bdunne new code pushed up, commit https://github.com/ManageIQ/manageiq-pods/pull/671/commits/464c84284efe94b0b85ece3cf432f2be0c5ba4d7 has the new logic for the secret watcher. I've added  predicate functions on the events to only trigger reconciliation on the httpd auth secret since we skip non-CR request reconciliations.